cli/command/container/cp.go

```diff
@@ -217,17 +224,16 @@ func copyFromContainer(ctx context.Context, dockerCli command.Cli, copyConfig cp
 		preArchive = archive.RebaseArchiveEntries(content, srcBase, srcInfo.RebaseName)
 	}
 
-	if !copyConfig.quiet {
-		fmt.Fprint(dockerCli.Out(), "Preparing to copy...\n")
-		fmt.Fprint(dockerCli.Out(), "\033[s")
+	if copyConfig.quiet {
+		return archive.CopyTo(preArchive, srcInfo, dstPath)
 	}
 
+	fmt.Fprint(dockerCli.Err(), "Preparing to copy...\n")
+	fmt.Fprint(dockerCli.Err(), "\033[s")
```

Comment:
This should probably be replaced with a `aec.XX`. I wasn't sure what code this was ☺️

Reply:
This cursor command (`\033[s`) saves the cursor position, in this case, the beginning the new line.

Reply:
I think that `aec.save()` could solve this.

Reply:
Hello @thaJeztah, how are you? Did you see this change in this line?

Reply:
oh! missed it (or thought you would include it in after this was merged); let me update